lib/conjur/base.rb

```diff
@@ -141,6 +141,24 @@ def new_from_key(username, api_key, remote_ip = nil)
       def new_from_token(token, remote_ip = nil)
         self.new.init_from_token token, remote_ip
       end
+
+      # Create a new {Conjur::API} instance from a file containing a token issued by the
+      # {http://developer.conjur.net/reference/services/authentication Conjur authentication service}
+      #
+      # This method is useful when an external process, such as a sidecar container, is continuously
+      # obtaining fresh tokens and writing them to a known file.
```

Comment:
Even though this paragraphs kinda sorta implies it, I feel it should be explicitly stated in this doc that the token is NOT read just once on calling this method, but that the path is remembered and continuously checked. 

Is it continuously? Or only on token expiration? It seems to be the latter. If so, maybe the whole expiration logic is unnecessary here. Just read the token every time one is needed. Or check if mtime is different than what was seen last. 

Another question: is the file read first time on init, or only when the token is actually needed for the first time?